css-positioning/absolute.css

```diff
@@ -0,0 +1,35 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 1rem;
+  right: 2rem;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  bottom: 0.5rem;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+  position: relative;
+}
```

Comment:
This needs `position: absolute` as well and either a `width / height` or you need to set all the offset (`top`, `left`, etc) properties to 0, so it will take up the full space of the image